seinfoflow

```diff
@@ -53,6 +53,9 @@ opts.add_argument("-w", "--min_weight", default=3, type=int,
                   help="Minimum permission weight.  Default is 3.")
 opts.add_argument("-l", "--limit_flows", default=0, type=int,
                   help="Limit to the specified number of flows.  Default is unlimited.")
+opts.add_argument("-c", "--conditionals", default=None,
```

Comment:
You're actually providing Boolean values, so the option should should be `-b`, `--booleans`